workspaces/ui-v2/src/pages/changelogHistory/ChangelogHistory.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { FC } from 'react';
+import { useHistory } from 'react-router-dom';
+import classNames from 'classnames';
+import { makeStyles } from '@material-ui/core/styles';
+
+import {
+  Link, // this should probably be one of our own global components
+} from '@material-ui/core';
+
+import {
+  Page,
+  useChangelogPages,
+  useDocumentationPageLink,
+} from '<src>/components';
+import { useBatchCommits } from '<src>/hooks/useBatchCommits';
+import { useSpacingStyles, useUtilityStyles } from '<src>/styles';
+import { formatTimeAgo } from '<src>/utils';
+
+export const ChangelogHistory: FC = () => {
+  const { loading, batchCommits } = useBatchCommits();
+  const changelogPage = useChangelogPages();
+  const documentationPage = useDocumentationPageLink();
+
+  const classes = useStyles();
+
+  return (
+    <Page>
+      <Page.Navbar />
+      <Page.Body padded className={classes.pageBody} loading={loading}>
+        <section className={classes.changelogSection}>
+          <h1>Changelog</h1>
+          <ol className={classes.commitsList}>
+            {batchCommits.map((batchCommit, i) => (
+              <li className={classes.commitsListItem} key={batchCommit.batchId}>
+                <Link
+                  className={classes.commitLink}
+                  href={
+                    i === 0
+                      ? documentationPage.linkTo()
+                      : changelogPage.linkTo(batchCommit.batchId)
+                  }
+                >
+                  <h4 className={classes.commitMessage}>
+                    {batchCommit.commitMessage}
+                  </h4>
+                  <span className={classes.commitTime}>
+                    {formatTimeAgo(new Date(batchCommit.createdAt))}
+                  </span>
+                </Link>
+              </li>
+            ))}
+          </ol>
+        </section>
```

Comment:
@niclim here's some of the changes I've made to the new page, representing fairly well how I think we should approach the new pages. See some of our private conversation about why I think going with semantic classnames is the better trade-off to make, especially from the perspective of working in tandem with a designer (and their tools). Happy to go into more detail if you'd like.